authorization-common-api/src/main/java/com/sequenceiq/authorization/resource/AuthorizationResourceAction.java

```diff
@@ -77,6 +77,9 @@
     DESCRIBE_RECIPE("environments/useSharedResource", AuthorizationResourceType.RECIPE),
     CREATE_RECIPE("environments/createRecipe", AuthorizationResourceType.RECIPE),
     DELETE_RECIPE("environments/deleteRecipe", AuthorizationResourceType.RECIPE),
+    DESCRIBE_CUSTOM_CONFIGS("datahub/describeCustomConfigs", AuthorizationResourceType.CUSTOM_CONFIGS),
```

Comment:
@horadla23 is this fine?

Reply:
@aakashshankar please refer here the related change in UMS code, based on that, I can tell if this will be OK from authz point of view

Reply:
https://github.infra.cloudera.com/thunderhead/thunderhead/pull/5613